framework/encode/vulkan_state_writer.cpp

```diff
@@ -188,9 +213,21 @@ uint64_t VulkanStateWriter::WriteState(const VulkanStateTable& state_table, uint
     marker.marker_type = format::kEndMarker;
     output_stream_->Write(&marker, sizeof(marker));
 
+    if (asset_file_stream_)
+    {
+        asset_file_stream_->Flush();
+    }
+
     // For the EndMarker meta command
     ++blocks_written_;
 
+    auto done = std::chrono::high_resolution_clock::now();
+    uint32_t time = std::chrono::duration_cast<std::chrono::milliseconds>(done - started).count();
+    GFXRECON_WRITE_CONSOLE("--------------------------------------")
```

Comment:
Should this and the other instance of timing data written to `GFXRECON_WRITE_CONSOLE` use `GFXRECON_LOG_INFO` instead?

Reply:
Turned these into `GFXRECON_LOG_INFO`